SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -454,11 +454,20 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing")
 	bool bEventTracingEnabled;
 
+	/*
+	 * -- EXPERIMENTAL --
+	 * Same as bEventTracingEnabled, but used if WITH_EDITOR is defined.
+	 */
+	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing")
```

Comment:
Wonder if we should make these private so that we enforce the use of GetEventTracingEnabled